container/common/helpers.go

```diff
@@ -155,6 +183,9 @@ func readUInt64(dirpath string, file string) uint64 {
 	if out == "" || out == "max" {
 		return 0
 	}
+	if out == "max" {
```

Comment:
This is impossible to hit, since we just checked `out == "" || out == "max"` above.

Reply:
ouch, sure.  Dropped it